docs/en/writing-running-appium/android_coverage.md

```diff
@@ -0,0 +1,292 @@
+# Android Coverage Guide
+## Requirement
+* Only Emulators or rooted phones are acceptable
+* Need to add UiAutomator2 support in your apps. That is to say, you need to implement a subclass of `Instrumentation`. `Instrumentation` will be responsible to collect your coverage. 
+* Need to implement a BroadcastReceiver to export coverage to disk files. This is because `Instrumentation` only collects data into memory.
+
+## Project Structure
+You may need the following structure of files:
+```
+src/main/java/com/example/pkg
+   |____ MainActivity.java *Your main activity*
```

Comment:
These items are still inconsistent. Some have an asterisk at the beginning and end of the description, some just the beginning, some not at all.